orders-war/src/main/java/au/gov/qld/pub/orders/entity/Item.java

```diff
@@ -67,14 +68,18 @@ protected Item() {
     
     public static Item createItem(ItemPropertiesDTO dto) {
         Item item = new Item();
-        BeanUtils.copyProperties(dto, item);
+        try {
+            BeanUtils.copyProperties(item, dto);
```

Comment:
with boon being replaced, beanUtils being used, the getter and setter's needed to be added to this class.